### DISCUSSION TITLE
Help install

### DISCUSSION THREAD

**Discussion by tammam050** [community/community#38085] [Other Feature Feedback, Questions, & Ideas]
Hi I'm new to this
Please help me and the correct steps to install
incomprehensible to me
https://github.com/urbanadventurer/Android-PIN-Bruteforce

thank you

**Comment by david-engelmann** [comment]
You might get better responses if you move this discussion into the issues section of the https://github.com/urbanadventurer/Android-PIN-Bruteforce repository
